kong/runloop/plugin_servers/init.lua

```diff
@@ -15,6 +17,75 @@ local rpc_notifications = {}
 --- currently running plugin instances
 local running_instances = {}
 
+local exposed_api = {
+  kong = kong,
+
+  ["kong.log.serialize"] = function()
+    local saved = save_for_later[coroutine.running()]
+    return cjson.encode(saved and saved.serialize_data or kong.log.serialize())
+  end,
+
+  ["kong.nginx.get_var"] = function(v)
+    return ngx.var[v]
```

Comment:
`ngx.var` is something we can cache at file level, this speed up this potentially hot call path a little considering variable access are notoriously slow in OpenResty already.

Reply:
this is to allow an external plugin (Go, JS) to do `ngx.var[]`;  caching could defeat the purpose and the performance issue is much less than the RPC overhead.